### ISSUE TITLE
uninitialized constant SimpleDelegator

### ISSUE THREAD

**Issue by aselder** [jnunemaker/httparty#531]
When running license_finder with httparty 0.15.2 we get this error. Reverting to httparty 0.13.7 fixes the issue.

```
/opt/circleci/.rvm/gems/ruby-2.2.6/gems/httparty-0.15.2/lib/httparty/response/headers.rb:3:in `<class:Response>': uninitialized constant SimpleDelegator (NameError)
	from /opt/circleci/.rvm/gems/ruby-2.2.6/gems/httparty-0.15.2/lib/httparty/response/headers.rb:2:in `<module:HTTParty>'
	from /opt/circleci/.rvm/gems/ruby-2.2.6/gems/httparty-0.15.2/lib/httparty/response/headers.rb:1:in `<top (required)>'
```

**Comment by aselder** [comment]
It looks like simple delegator doesn't exist in Ruby 2.2.6 at least:
```
 ~  irb
 :001 > SimpleDelegator
NameError: uninitialized constant SimpleDelegator
	from (irb):1
	from /Users/aselder/.rvm/rubies/ruby-2.2.6/bin/irb:11:in `<main>'
```

**Comment by aselder** [comment]
PR coming shortly

**Comment by aselder** [comment]
Closed with #532

**Comment by luckyraul** [comment]
@jnunemaker release please
